docs/profiler.md

```diff
@@ -0,0 +1,122 @@
+# What is the profiler tool
+
+The major goal of profiler tool is to provide an estimation how fast can be the proof created.
+Creating a proof in spacemesh depends on few major factors:
+* CPU speed
+* Disk speed
+* Amount of storage initialized
+
+The major task of the profiler tool is to estimate how fast can be the proof created for a given CPU speed and disk speed.
+
+The best way to get the profiler tool is to get it from the [releases page](https://github.com/spacemeshos/post-rs/releases).
+
+It's very important to remember that the proof time itself does not need to be optimized against, we just need to generate valid proof in time which on the mainnet is 12h.
```

Comment:
Can you explain what you mean with this? I assume that it is not necessary to optimize proofing speed as long as the proof can be generated in the necessary time window?

Reply:
Sure, you don't need to complete the proof asap, you want to be as much to the end as possible, and as much as it makes sense in MOST cases that will mean the lowest energy consumption.